recipes/junit2html/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "junit2html" %}
+{% set version = "022" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 2160977bec4168babd323da4cf86f5b33ae0220007df0f94abc3f0a57e6d9cdd
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+
+test:
+  imports:
+    - junit2htmlreport
+    - junit2htmlreport.matrix
+    - junit2htmlreport.merge
+    - junit2htmlreport.parser
+    - junit2htmlreport.runner
+    - junit2htmlreport.tag
+    - junit2htmlreport.textutils
+
+about:
+  home: https://gitlab.com/inorton/junit2html
+  license: "OSI Approved :: MIT"
```

We generally prefer the SPDX identifiers for consistency:

```suggestion
  license: MIT
```